lib/GADS/Role/Presentation/Datum/Curval.pm

```diff
@@ -0,0 +1,33 @@
+package GADS::Role::Presentation::Datum::Curval;
+
+use Moo::Role;
+
+sub _presentation_details {
+    my $self = shift;
+
+    return [] unless $self->as_string;
+
+    my $rti = $self->column->refers_to_instance;
+
+    my @links = map +{
+        id   => $_->{id},
+        href => $_->{value},
+        refers_to_instance => $rti,
+    }, @{$self->_text_all};
+
+    return \@links;
+}
+
+sub presentation {
+    my $self = shift;
+
+    return {
+        type  => $self->column->type,
+        text  => $self->text,
+        id    => $self->id,
```

Comment:
I don't think it's used in there - the only id I can see is the id of the link, not the id of the datum.

This is the problem code: https://github.com/ctrlo/GADS/blob/8e1f4cac06404a23c4306e0a028fb11d2dab6633/lib/GADS/Datum/Curval.pm#L204

I want to remove the single "id" properties from datums as they are no longer applicable with the new multivalue feature. This is an interim fix.